src/components/radios/index.md.njk

```diff
@@ -84,19 +84,23 @@ If one or more of your radio options is different from the others, it can help u
 
 ### Conditionally revealing content
 
-You can conditionally reveal content when the user selects a particular radio, so they only see content when it’s relevant to them.
+You can conditionally reveal a related question when the user selects a particular radio option, so they only see content when it’s relevant to them.
 
-For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
+This might make two related questions easier to answer by grouping them together. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
```

Comment:
```suggestion
This might make two related questions easier to answer by grouping them on the same page. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
```